analyzer/analyzer.go

```diff
@@ -217,6 +217,7 @@ func Analyze(ctx context.Context, client *containerd.Client, ref string, opts ..
 			if err := rc.Record(path); err != nil {
 				log.G(ctx).WithError(err).Debugf("failed to record %q", path)
 			}
+			log.G(ctx).Debugf("success record path %q", path)
```

Comment:
You can get the full list of recorded path using `--record-out` option.

Instead of printing all paths, can we just print the `succeeded` message with the number of recorded path just before this `go func` returns?

Reply:
Done, do u think we should add `use with --record-out option to see details` with the log?